### ISSUE TITLE
`/create-a-course/` LP copy feedback

### ISSUE THREAD

**Issue by niranjan-uma-shankar** [Automattic/wp-calypso#93020]
### Quick summary

This came up in the dotcom walkthrough p58i-hK7-p2. 

* This CTA copy doesn't look apt
<img width="475" alt="Screenshot 2024-07-26 at 5 13 17 PM" src="https://github.com/user-attachments/assets/f12e007f-ac1b-43f6-a50e-2b50cbbe038f">

* It was mentioned that the copy in this section is not selling
<img width="761" alt="Screenshot 2024-07-26 at 5 14 04 PM" src="https://github.com/user-attachments/assets/f4958248-19f8-43e5-8228-a5f8b5b76a20">

**Comment by michaelpick** [comment]
Hey @niranjan-uma-shankar! Just saw this (I was out Friday and Monday). 

Do you happen to know who we should share copy with for sign-off and/or implementation? I had a look at the video and it sounds like we'd like to change the section above to be closer inline with the more benefits, less features-driven approach of the Sensei LMS site. There was mention of just calling out the plugin not being very inspired, which I can't disagree with! :) 

We'll take a look ASAP.l

**Comment by alvezfrancese** [comment]
Hi there @niranjan-uma-shankar! We took a look and iterated the copy [here](https://docs.google.com/document/d/1cONyU2Yjs3xBdk9ptJemmSiMLJq7wWv6FmUy_nOpSfY/edit#heading=h.2yav0jxosyxa). WDYT? 

cc. @michaelpick

**Comment by niranjan-uma-shankar** [comment]
Thanks. I'll defer to @ianstewart who raised this in the walkthrough video.

**Comment by niranjan-uma-shankar** [comment]
Was the feedback given to the hero section to the "Powered by Sensei LMS" section? I assumed it was the latter but I might be wrong.

**Comment by alvezfrancese** [comment]
You're right: it was the Powered by Sensei LMS. We also iterated the hero just in case.

**Comment by ianstewart** [comment]
Left a comment in  the feedback doc. The use of "Course" (captialized, name of Theme) threw me off in my walkthrough. "Create a new site" or "Get started" seem more appropriate.
